### ISSUE TITLE
will the certificate issued by cfssl be trusted by latest chrome or firefox?

### ISSUE THREAD

**Issue by 8090s** [cloudflare/cfssl#1184]
hello,
I have a question, will the certificate issued by cfssl be trusted by latest chrome or firefox?

**Comment by ClintRajaniemi** [comment]
No, you will need to import the root CA and any intermediates into your browser. Others won't trust your root CA (and hence will not trust your other certificates). This is for an internal PKI where you control the hosts that will trust these certs. For other use cases, you'll need to get a trusted root/intermediate CA to sign a certificate on your behalf to get others to trust your certificate.
